packages/Heading/src/Heading.styles.js

```diff
@@ -1,7 +1,9 @@
 import tokens from "@paprika/tokens";
 import stylers from "@paprika/stylers";
 
-const headingStyles = () => `
+// default heading styles
```

Comment:
We don't need comments if it is self-explanatory

Reply:
done